yarf/rf_libraries/resources/kvm.resource

```diff
@@ -340,3 +340,35 @@ Ensure ${destination} Does Not Match
     IF    ${match}
         Fail                    Found ${destination} where it wasn't intended or expected
     END
+
+Drag ${target} And Drop On ${destination}
+    [Documentation]    Move the pointer to a target, drag and drop it on a destination.
+    ...
+    ...    Embedded arguments:
+    ...    - ${target}: Can be a string, or the path of an image template file representing
+    ...    the item to drag or a coordinate tuple (x, y) of integers representing the
+    ...    absolute position to drag from. For details please see `Move Pointer to ${destination}`.
+    ...    - ${destination}: Can be a string, or the path of an image template file representing
+    ...    the target location to drop on, or a coordinate tuple (x, y) of integers representing the
+    ...    absolute position to drop on. For details please see `Walk Pointer To ${destination}`.
+    ...
+    [Arguments]             ${step_distance}=16     ${delay}=0.01
```

Comment:
```suggestion
    [Arguments]             ${step_distance}=0.01     ${delay}=0.01
```

We know step_distance should be in [0,1], why having a default out of boundaries?

Reply:
Err... I think we are calling `walk_pointer_to_absolute()` down the line here and we convert the step distance to [0,1] domain there.

Reply:
ah that is not fixed yet on main